### DISCUSSION TITLE
Please continue development on this plugin

### DISCUSSION THREAD

**Discussion by nihitkhandelwal-eng** [st3v3nmw/obsidian-spaced-repetition#348] [General]
I have been using RemNote for the past 1 year for my academic notes & the only reason I use RemNote for that is the powerful flashcards & spaced repetition system.

If development on this plugin continues, it would enable many like me to finally migrate from RemNote to Obsidian & will be happy to support development.
There are many pending issues here & I am not sure if this plugin is maintained.

Thanks a lot for this plugin & I hope you continue development on this.

**Comment by st3v3nmw** [comment]
There's a new release available (v1.7.0). It took a while but...
